projects/packages/jitm/package.json

```diff
@@ -29,5 +29,8 @@
 	"engines": {
 		"node": "^14.16.0",
 		"yarn": "^1.3.2"
+	},
+	"volta": {
```

Comment:
I don't know if we'd want to extend here (and for the other packages), since the JITM package is also available standalone, in a situation where there is nothing to extend:
https://github.com/Automattic/jetpack-jitm/blob/master/package.json

Reply:
If nothing else, we could copy the configuration and use a script (run from precommit and CI) to ensure that the copies remain in sync and are in sync with .nvmrc. We could also verify that the "engines" are in sync everywhere too, and that the versions are compatible.

Or we could wait for volta to resolve https://github.com/volta-cli/volta/issues/282 so we could just drop a .voltarc or whatever in the monorepo root.

Reply:
> I don't know if we'd want to extend here (and for the other packages), since the JITM package is also available standalone...

Makes sense! I'd forgotten about the mirror repos. I've updated the PR so that each package.json has the version info rather than extending the root, except for `projects/plugins/jetpack/tests/e2e/package.json`, which now extends `projects/plugins/jetpack/package.json`.

Reply:
> We could also verify that the "engines" are in sync everywhere too, and that the versions [in .nvmrc versus package.json "engines.node"] are compatible.

FYI, I pushed #19748 to ensure that `engines` in package.json is in sync everywhere, and #19750 to check that the version in `.nvmrc` satisfies that called for by package.json.

Once those are merged, it should be straightforward to do the same for the volta blocks.